src/test/java/StringMethodTest.java

```diff
@@ -0,0 +1,82 @@
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+@DisplayName("String 클래스에 대한 학습 테스트")
+public class StringMethodTest {
+
+    @DisplayName("쉼표로 split 했을 때 1과 2로 잘 분리되는지 확인한다")
+    @Test
+    void splitWithCommaForTwoElementsTest() {
+        // Given
+        String input = "1,2";
+
+        // When
+        String[] actual = input.split(",");
+
+        // Then
+        assertThat(actual)
+                .as("\"1,2\"을 ,로 split 했을 때 1과 2로 잘 분리되는지 확인한다")
+                .containsExactly("1", "2");
+    }
+
+    @DisplayName("쉼표로 split 했을 때 1만 포함하는 배열이 반환되는지 확인한다")
+    @Test
+    void splitWithCommaForOneElementTest() {
+        // Given
+        String input = "1";
+
+        // When
+        String[] actual = input.split(",");
+
+        // Then
+        assertThat(actual)
+                .as("\"1\"을 ,로 split 했을 때 1만 포함하는 배열이 반환되는지 확인한다")
+                .contains("1");
```

Comment:
```suggestion
        assertThat("1".split(",")).contains("1");
```

위와 같이 극단적으로 단순하게 구현하는 것은 어떨까?
지금 구현과 장단점은 뭘까?

Reply:
- 테스트의 의도는 DisplayName 이나 메서드명으로 충분히 표현할 수 있기에 as 로 Description 을 표현하지 않아도 되지 않을까 생각됩니다.

### 지금 구현과의 비교
- 장점 : 우선 코드의 양이 적어지고 한눈에 어떤 의미인지 알 수 있다.
- 단점 : 확장성이 다소 떨어질 수 있다.


이 정도가 우선 생각납니다.